lib/release/release-manager.js

```diff
@@ -194,63 +217,71 @@ class ReleaseManager {
 
   #getReleaseSteps () {
     const R = `-R ${this.#owner}/${this.#repo}`
-    /* eslint-disable max-len */
-    const MANUAL_PUBLISH_STEPS = dedent`
+
+    const manualSteps = `
       1. Checkout the release branch and test
   
-          \`\`\`sh
+          ${block('sh')}
           gh pr checkout <PR-NUMBER> --force
           npm ${this.#lockfile ? 'ci' : 'update'}
           npm test
           gh pr checks <PR-NUMBER> ${R} --watch
-          \`\`\`
+          ${block()}
   
       1. Publish workspaces
   
-          \`\`\`sh
+          ${block('sh')}
           npm publish -w <WS-PKG-N>
-          \`\`\`
+          ${block()}
   
       1. Publish
   
-          \`\`\`sh
+          ${block('sh')}
           npm publish <PUBLISH-FLAGS>
-          \`\`\`
+          ${block()}
   
       1. Merge release PR
   
-          \`\`\`sh
-          gh pr merge <PR-NUMBER> ${R} --rebase
+          ${block('sh')}
+          gh pr merge <PR-NUMBER> ${R} --squash
           git checkout <BASE-BRANCH>
           git fetch
           git reset --hard origin/<BASE-BRANCH>
-          \`\`\`
+          ${block()}
     `
 
-    const AUTO_PUBLISH_STEPS = dedent`
+    const autoSteps = `
       1. Approve this PR
   
-          \`\`\`sh
+          ${block('sh')}
           gh pr review <PR-NUMBER> ${R} --approve
-          \`\`\`
+          ${block()}
   
       1. Merge release PR :rotating_light: Merging this will auto publish :rotating_light:
   
-          \`\`\`sh
-          gh pr merge <PR-NUMBER> ${R} --rebase
-          \`\`\`
+          ${block('sh')}
+          gh pr merge <PR-NUMBER> ${R} --squash
```

Comment:
I learned that release-please will work faster if the release PRs are merged with squash instead of rebase.